src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/prisonsapi/InmateDetail.kt

```diff
@@ -13,8 +15,8 @@ data class AssignedLivingUnit(
   val agencyName: String,
 )
 
-enum class InOutStatus {
-  IN,
-  OUT,
-  TRN,
+enum class InmateStatus(@JsonValue val value: String) {
+  IN("ACTIVE IN"),
+  OUT("INACTIVE OUT"),
+  TRN("INACTIVE TRN"),
```

Comment:
Looking at the OpenAPI spec for the Prison API, the only possible values are `ACTIVE IN`, `ACTIVE OUT` or `INACTIVE TRN`:

![image](https://github.com/ministryofjustice/hmpps-approved-premises-api/assets/109774/11fff525-cfe0-40d2-8aab-8378ce39f73d)

Reply:
I was looking at this repo: https://github.com/ministryofjustice/prison-api/blob/main/src/main/java/uk/gov/justice/hmpps/prison/api/model/InmateDetail.java

I believe the swagger doc is not giving full pic.

Reply:
Ah, I get ya. I think I'd rather cast it to an enum than a string, because passing the data around will be more predictable and less brittle, but I'll check with the Prison team what all the expected values _should_ be.